src/components/ContentPreview/__tests__/ContentPreview-test.js

```diff
@@ -791,6 +791,82 @@ describe('components/ContentPreview/ContentPreview', () => {
             });
             expect(wrapper.state('currentFileId')).toBe(newFileId);
         });
+
+        test('should update preview if navigation occurs then browser back clicked', () => {
+            // navigation
+            wrapper.setState({
+                currentFileId: newFileId,
+            });
+            expect(wrapper.state('prevFileId')).toBe(initialFileId);
+            expect(wrapper.state('currentFileId')).toBe(newFileId);
+
+            // URL update
+            wrapper.setProps({
+                fileId: newFileId,
+                foo: 'baz',
+            });
+            expect(wrapper.state('currentFileId')).toBe(newFileId);
+            expect(wrapper.state('prevFileId')).toBe(newFileId);
```

Comment:
I may be out of the loop here, but shouldn't the "previous id" by set to `initialFileId` here? Same question below.

Reply:
No, I think maybe the variable name is confusing:
```
prevFileId?: string, // the prop "fileId" which is needed to implement getDerivedStateFromProps
```

lets walk through a simple example of file id 1 and 2

render file id 1:
```
props: 
fileId: 1

prev state: null

new state: 
prevFileId: 1,
currentFileId:1
```

user navigates to file id 2 via navigation arrows:
```
props: 
fileId: 1

prev state:
prevFileId: 1,
currentFileId: 1

new state: 
prevFileId: 1,
currentFileId: 2
```

application listens to navigate event, updates url, and updates fileId prop to 2:
```
props:
fileId: 2

prev state: 
prevFileId: 1,
currentFileId: 2

new state: 
prevFileId: 2,
currentFileId: 2
```

user clicks back button to file id 1:
```
props:
fileId: 1

prev state: 
prevFileId: 2,
currentFileId: 2

new state:
prevFileId: 1,
currentFileId: 1
```

Reply:
I see, so it's a snapshot of the previous state, rather than the previous file. Fair enough, though a little confusing at first glance.

Reply:
Yeah, good feedback. I renamed it in my latest commit to hopefully make more sense 👍